spec/lib/data_update_scripts/migrate_data_to_work_field_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb",
+)
+
+describe DataUpdateScripts::MigrateDataToWorkField, sidekiq: :inline do
+  let!(:user) { create(:user) }
+  let(:profile) { user.profile }
+
+  before do
+    ProfileField.find_or_create_by!(label: "Work", input_type: :text_field)
+    Profile.refresh_attributes!
+  end
+
+  it "migrates employment titles without employer name" do
+    profile.update!(employment_title: "Tester")
+    expect { described_class.new.run }
+      .to change { profile.reload.work }.from(nil).to("Tester")
+  end
+
+  it "migrates employment titles and employer name" do
+    profile.update!(employment_title: "Tester", employer_name: "ACME Inc.")
+    expect { described_class.new.run }
+      .to change { profile.reload.work }.from(nil).to("Tester at ACME Inc.")
+  end
+
+  it "ignores blank employment titles" do
+    profile.update!(employment_title: "", employer_name: "ACME Inc.")
+    expect { described_class.new.run }.not_to change { profile.reload.work }
+  end
+
+  it "ignores blank employer names" do
```

Comment:
does this case belong there? It looks like the body is identical to "migrates employment titles without employer name"  but the description suggests nothing happens (description text and test code conflict).

I think this it block can just be deleted safely with no difference in coverage.

Reply:
Yes, that was the intention, if the employer name is missing, we produce exactly the same result, no trailing whitespace, etc. It was quite useful while writing the code (yes, I really did write the tests first).